packages/realm-react/src/__tests__/useObjectRender.test.tsx

```diff
@@ -438,6 +438,24 @@ describe("useObject: rendering objects with a Realm.List property", () => {
       fireEvent.press(rerenderButton);
       expect(objectChangeCounter).toHaveBeenCalledTimes(expectedCount);
     });
+    it("will rerender when the realm database is cleared", async () => {
+      await setupTest();
+
+      const initialCount = 2;
+
+      expect(objectChangeCounter).toHaveBeenCalledTimes(initialCount);
+
+      await act(async () => {
+        testRealm.write(() => {
+          testRealm.deleteAll();
+        });
+        forceSynchronousNotifications(testRealm);
+      });
+
+      await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Out of curiosity, what do we need the timeout for here? Is it to ensure the listeners have been called?

Reply:
Oh, I can probably remove that.  I was having problems getting the tests to pass and I thought it was a timing issue.  In the end I was looking at the wrong mock function to base my counts off of.